demo/lightning/image-segmentation/train.py

```diff
@@ -1,14 +1,12 @@
 import lightning as pl
 
-from model import Unet3D
+from model import Unet3DLightning
 from data import Unet3DDataModule
+from arguments import PARSER
 
 if __name__ == "__main__":
-    model = Unet3D()
-    train_data_loader = Unet3DDataModule()
-    trainer = pl.Trainer(
-        devices=2, 
-        accelerator="gpu", 
-        max_epochs=5
-    )
+    flags = PARSER.parse_args()
+    model = Unet3DLightning(flags)
+    train_data_loader = Unet3DDataModule(flags)
+    trainer = pl.Trainer(accelerator="cpu", max_epochs=5)
```

Comment:
Can you make these arguments configurable?

Reply:
Done! Added a new argument `accelerator` which can be set to the accelerator type. It defaults to `cpu`.